load/loader.py

```diff
@@ -102,7 +102,16 @@ def load_voting_history(opts, conf):
         '-param:reportFileDir={}'.format(opts.input_file),
         '-param:reporterKey={}'.format(opts.reporter_key)
     ])
-
+    elif opts.state == 'ny':
+        fl_path = os.path.dirname(os.path.join(conf['data_path'],
```

Comment:
Actually, would it be possible to standardize the `reportFile` vs `reportFileDir` param across both of these? That way, since the file is the same we could just check if the state is either Florida or New York, then load the path from the abbreviation without duplicating a lot of it